plugins/modules/aws_kms_info.py

```diff
@@ -468,11 +466,11 @@ def main():
     ret_params = dict(kms_keys=filtered_keys)
 
     # We originally returned "keys"
-    if module.params['keys_attr']:
+    if module.params.get('keys_attr') is not None:
         module.deprecate("Returning results in the 'keys' attribute conflicts with the builtin keys() method on "
-                         "dicts and as such is deprecated and is now ignored. Please use the kms_keys attribute. This warning can be "
-                         "silenced by setting keys_attr to False.",
-                         version='3.0.0', collection_name='community.aws')
+                         "dicts and as such was removes in version 3.0.0.  Please use the kms_keys attribute. "
```

Comment:
```suggestion
                         "dicts and as such was removed in version 3.0.0.  Please use the kms_keys attribute. "
```
Nit, otherwise LGTM